.github/workflows/ci.yaml

```diff
@@ -45,29 +44,29 @@ jobs:
     runs-on: ${{ matrix.platform }}
 
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v3
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         env:
           servicex_version: 1.0a1
         run: |
           python -m pip install --upgrade pip setuptools wheel
-          pip install --no-cache-dir -e .[test]
-          pip list
+          python -m pip install --no-cache-dir -e .[test]
+          python -m pip list
       - name: Test with pytest
         run: |
           python -m pytest
       - name: Code coverage with pytest
-        if: github.event_name == 'push' && matrix.python-version == 3.7 && matrix.platform == 'ubuntu-latest'
+        if: github.event_name == 'push' && matrix.platform == 'ubuntu-latest'
```

Comment:
`matrix.python-version == 3.7` was mismatched with the Python runtime and so was never running. There's no need to specify the runtime as only one Python version is running this, so remove to simplify and make it run.